camelot/core/naming.py

```diff
@@ -536,28 +750,22 @@ def __init__(self):
         # resolution of subcontexts.
         self._name = tuple()
 
-    def new_context(self):
-        return NamingContext()
+        # Add immutable bindings for constants' values and contexts for each supported 'constant' python type.
+        constants = self.bind_new_context('constants', immutable=True)
+        for constant_type in (str, int, Decimal): # Do not support floats, as vFinance uses Decimals throughout
+            constants.bind_context(constant_type.__name__.lower(), ConstantNamingContext(constant_type), immutable=True)
+        constants.bind('null', None, immutable=True)
+        constants.bind('true', True, immutable=True)
+        constants.bind('false', False, immutable=True)
```

Comment:
not sure, maar mss moeten we ook nog het 'speciale' Any object binden, voor gebruik in de filters